sktime/utils/validation/panel.py

```diff
@@ -108,17 +112,18 @@ def check_X(
 
 def check_y(y, enforce_min_instances=1, coerce_to_numpy=False):
     """Validate input data.
+
     Parameters
     ----------
     y : pd.Series or np.array
     enforce_min_instances : int, optional (default=1)
         Enforce minimum number of instances.
     coerce_to_numpy : bool, optional (default=False)
-        If True, y will be coerced to a numpy array.
+         If True, y will be coerced to a numpy array.
 
     Returns
     -------
-    y : pd.Series or np.array
+    y : pd.Series
```

Comment:
should this not be: `pd.Series` if `coerce_to_numpy=True`, `np.array` if `coerce_to_numpy=False`

Reply:
Thank you @fkiraly . I will make necessary corrections.